lib/handler/status.c

```diff
@@ -236,9 +234,47 @@ static int on_req_json(struct st_h2o_status_handler_t *self, h2o_req_t *req)
     return 0;
 }
 
+static h2o_iovec_t *build_status_list(const char *param, size_t len)
```

Comment:
How about calling `h2o_contains_token` within `on_req_json` for each of the status handlers, instead of building a list of selected names using this function?